cli/src/pcluster/validators/awsbatch_validators.py

```diff
@@ -195,3 +195,15 @@ def _get_supported_architectures_for_instance_type(instance_type):
     def _is_instance_type_format(candidate):
         """Return a boolean describing whether or not candidate is of the format of an instance type."""
         return re.search(r"^([a-z0-9\-]+)\.", candidate) is not None
+
+
+class FsxSchedulerValidator(Validator):
+    """
+    Validator for FSx and scheduler.
+
+    Fail if using AWS Batch and FSx for Lustre, not supported yet.
+    """
+
+    def _validate(self, scheduler: str):
```

Comment:
This validator is inside `awsbatch_validators.py` file while expecting the a parameter `scheduler` that can be "slurm" or "awsbatch". It might be better to put this validator to fsx validators file